src/drt/src/dr/FlexDR_init.cpp

```diff
@@ -1019,12 +1019,26 @@ void FlexDRWorker::initNet_termGenAp_new(const frDesign* design, drPin* dPin)
             xLoc = *(xLocs.begin());
           } else {
             xLoc = (xl(pinRect) + xh(pinRect)) / 2;
+            frCoord manuGrid = getTech()->getManufacturingGrid();
```

Comment:
There look to be two similar blocks in the frcBTerm section below.

Reply:
If you are going to have four copies of this it might be time to factor it into a helper function.